src/DIRAC/RequestManagementSystem/private/OperationHandlerBase.py

```diff
@@ -141,8 +141,8 @@ def setOperation(self, operation):
             raise TypeError("expecting Operation instance")
         self.operation = operation
         self.request = operation._parent
-        self.log = gLogger.getSubLogger(
-            "pid_%s/%s/%s/%s" % (os.getpid(), self.request.RequestName, self.request.Order, self.operation.Type)
+        self.log = gLogger.getLocalSubLogger(
```

Comment:
Are you sure that nowhere in operations we try to get a sublogger from that?

Reply:
Actually there is one place T_T
https://github.com/DIRACGrid/DIRAC/blob/ed32a3c5a6cbf8fabab1f6f1f780267acfb2148d/src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py#L63